dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -115,81 +134,84 @@
                             </VisualStateGroup>
                         </VisualStateManager.VisualStateGroups>
                         <StackPanel Orientation="Horizontal"
-                                    Grid.Row="0"
                                     Grid.Column="0"
                                     HorizontalAlignment="Right">
                             <Button x:Name="FirstPageButton"
-                                    IsEnabled="False"
-                                    Style="{StaticResource NavigationBackButtonNormalStyle}"
+                                    Style="{TemplateBinding FirstButtonStyle}"
                                     FontFamily="{StaticResource SymbolThemeFontFamily}"
                                     Content="&#xE892;"/>
-                            <TextBlock x:Name="FirstPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
-                                        Text="{TemplateBinding FirstPageButtonText}"
-                                        Visibility="{Binding ElementName=FirstPageButton, Path=Visibility}"/>
                             <Button x:Name="PreviousPageButton"
-                                    IsEnabled="False"
-                                    Style="{StaticResource NavigationBackButtonNormalStyle}"
+                                    Style="{TemplateBinding PreviousButtonStyle}"
                                     FontFamily="{StaticResource SymbolThemeFontFamily}"
                                     Content="&#xE76B;"/>
-                            <TextBlock x:Name="PreviousPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
-                                        Text="{TemplateBinding PreviousPageButtonText}"
-                                        Visibility="{Binding ElementName=PreviousPageButton, Path=Visibility}"/>
                         </StackPanel>
-                        <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal" Grid.Row="0" Grid.Column="1">
-                            <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{TemplateBinding PrefixText}" />
+                        
+                        <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal"
+                                Grid.Column="1">
+                            <TextBlock x:Name="PrefixTextLabel" Text="{TemplateBinding PrefixText}" VerticalAlignment="Center" Margin="0,0,5,0"/>
                             <!--NumberBox Display-->
-                            <controls:NumberBox x:Name="NumberBoxDisplay" HorizontalAlignment="Center" VerticalAlignment="Center"
-                                            Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=NumberOfPages}" Minimum="1"
-                                            Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=SelectedIndex, Mode=TwoWay}"
```

Comment:
>="{ [](start = 49, length = 3)

Why is the value binding removed?

Reply:
Because the value binding wouldn't reflect what we want to do. Everything the user enters is 1 based, so the smallest they can enter is 1, the largest is NumberOfPages. The PagerControl.SelectedPageIndex property is base 0, so it goes from 0 to NumberOfPages-1. The prototype treated the SelectedPageIndex base 1, that's why the binding worked in that case.